package.json

```diff
@@ -56,6 +56,7 @@
     "babel-plugin-transform-runtime": "6.23.0",
     "babel-preset-env": "1.6.1",
     "babel-preset-react": "6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
```

Comment:
You don't seem to be using this anywhere? Is it required?

Reply:
Nupe that was part of the experiment 😛